envs/README.md

```diff
@@ -0,0 +1,157 @@
+
+```commandline
+conda env create -f ENVIRONMENT.yml
+```
+
+> **Note**: Instead of `conda` you can use `mamba` to speed up installation
+
+### Python environments
+
+There are multiple different environments for the python dependencies:
+
+| YAML file location       | Environment name     | Description                                                             |
+|--------------------------|----------------------|-------------------------------------------------------------------------|
+| `scib-pipeline-R4.0.yml` | `scib-pipeline-R4.0` | Base environment for calling the pipeline using R 4.0 and python >= 3.7 |
+| `scib-pipeline-R3.6.yml` | `scib-pipeline-R3.6` | Base environment for calling the pipeline using R 3.6 and python >= 3.7 |
+| `scIB-python-paper.yml`  | `scIB-python-paper`  | Environment used for the results in the [publication][publication]      |
+
+The `scib-pipeline` environment is the one that the user activates before calling the pipeline. It needs to be specified
+under the `py_env` key in the config files under `configs/` so that the pipeline will use it for running python methods.
+Alternatively, you can specify `scIB-python-paper` as the `py_env` to recreate the environment used in the paper to
+reproduce the results.
+
+> **Note**: `scIB-python-paper` is deprecated and only included for reproducibility purposes and is not updated with
+> changes to the pipeline code.
+> We don't recommend using it, as old dependencies tend to break after newer packages are available.
+
+### R environments
+
+| YAML file location       | Environment name     | R dependency file        | Description                                                        |
+|--------------------------|----------------------|--------------------------|--------------------------------------------------------------------|
+| `scib-R4.0.yml`          | `scib-R4.0`          | 'dependencies-R4.0.tsv'  | R environment using R 4.0                                          |
+| `scib-R3.6.yml`          | `scib-R3.6`          | 'dependencies-R3.6.tsv'  | R environment using R 3.6                                          |
+| `scIB-R-integration.yml` | `scIB-R-integration` | 'r36_dependencies.tsv'   | Environment used for the results in the [publication][publication] |
+
+Depending on the R environment used, some R packages must be additionally installed in R instead of conda. For
+convenience, we provide the `install_R_methods.R` scripts that installs the necessary dependencies through R
+directly. Don't forget to set the environment variables before installing anything through R. e.g. for `scib-R`:
+
+> **Note**: `scIB-python-paper` is deprecated and only included for reproducibility purposes and is not updated with
+> changes to the pipeline code.
+> We don't recommend using it, as old dependencies tend to break after newer packages are available.
+
+## 2. Setting Environment Variables
```

Comment:
Luke posted this conda package to do this automatically in the tools channel I think. Would this be an option here?

Reply:
I haven't tried it yet. Might be something for a new PR though.